src/encoded/static/components/datacolors.js

```diff
@@ -221,7 +221,14 @@ class DataColors {
     //         integer -- do not pass a string with a "%" sign at the end.
     colorList(keys, options) {
         let colors = [];
-        if (keys && keys.length > 0) {
+        if (options && keys && options.merryGoRoundColors && keys.length > 0) {
```

Comment:
merryground gives leeway in that the colors wrap back to the first one instead of showing default, after the color-limit is reached. I can see if you say it is YAGNI-violation.

Reply:
Good point. Done.